### DISCUSSION TITLE
How to truncate setup minimum charachters for a file name downloaded ?

### DISCUSSION THREAD

**Discussion by hitzhub0** [JunkFood02/Seal#1685] [Q&A]
I download few videos a day from FB for offline reference. 
But when those videos are downloaded file names are way too long, how can i setup minimum file length with only required information ?

**Comment by JunkFood02** [comment]
settings  - general - restrict filenames

**Reply by hitzhub0** [reply]
Thanks for reply,
Cannot locate that option there?
Using latest
![Screenshot_2024-09-18-18-35-45-602_com junkfood seal preview](https://github.com/user-attachments/assets/f72a6c45-5423-4b2c-8ce3-10c30cb980c8)
![Screenshot_2024-09-18-18-35-41-320_com junkfood seal preview](https://github.com/user-attachments/assets/ea24b727-7583-489a-b924-c323aa243937)
 seal preview

**Reply by JunkFood02** [reply]
oops sorry it should be under "download directory"
